python/docs/source/_static/versions.json

```diff
@@ -0,0 +1,278 @@
+[
```

Comment:
I wonder if we better remove EOL releases ... but no strong opinion WDYT @srowen and @dongjoon-hyun ?

Reply:
Yes, let's just start with latest versions even, as a convenience to switch. People can always find older versions manually, but we don't need to make them extra discoverable

Reply:
Done.

Reply:
We have to remember to update this with new releases - I wonder, do we have another document to add that step to?

Also I think we could even just retain the latest versions in each release branch?

Reply:
Yeah, I will add this logic to `release-build. sh`, which is what I want to do in the next PR, 😄